CoinDock-api/app/Models/V1/User.php

```diff
@@ -272,6 +279,29 @@ public function signUp()
         return $this->hasOne(Signup::class);
     }
 
+    public function updateProfile(updateProfileRequest $request, User $user)
+    {
+        $updatedUser = $request->all();
+        $user->update($updatedUser);
+        return response([
+            'message' => 'Profile updated succesfully',
+            'results'=>[
+                'user'=>new UserResource($user)
+            ]
+        ],Response::HTTP_OK );
+    }
+
+    public function changePassword(updatePasswordRequest $request, User $user)
+    {
+        $updatedPassword = $request->password;
+        
+        $this::whereId($user->id)->update(['password' => bcrypt($updatedPassword)]);
```

Comment:
Are we bcrypt() before storing the password?

And the above function was in the user model. so no need to add whereId() check and $user param.

Reply:
done